sourcegraph/about
# Bug Fix: Follow up for anchor linked sections

This is a follow up PR for #5637 and provides a much simpler solution that works for inbound linked sections and internal linked sections from page to page. This closes #5732.

### Changelog
- removed hook
- added css selector

### Test

1. Ensure prettier has standardized the proposed changes.
2. Check some anchor links (ie: on `/terms/cloud`) by clicking some anchor links and also loading an anchor link in a new tab
